.gitignore

```diff
@@ -2,5 +2,7 @@
 .sass-cache/
 bower_components/
 node_modules/
+demos/
```

Comment:
Because when I run obt demo --local I get the below tree, this gitignore
config will ignore everything in red:

demos
├── element-tracking.html
├── local
│   ├── element-tracking.css
│   ├── element-tracking.html
│   └── element-tracking.js
└── src

    ├── config.json

    ├── element-tracking.js

    ├── element-tracking.mustache

    └── element-tracking.scss

On 25 March 2015 at 10:46, Alberto Elias <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/Financial-Times/o-viewport/pull/17#discussion_r27109867>
> :
>
> > @@ -2,5 +2,7 @@
> >  .sass-cache/
> >  bower_components/
> >  node_modules/
> > +demos/
>
> Why are you ignoring demos? You should just be ignoring demos/local
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-viewport/pull/17/files#r27109867>.
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.